static/src/javascripts/projects/common/modules/experiments/ab-ui.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+
 import template from 'lodash/template';
 import { local as storage } from 'lib/storage';
 import $ from 'lib/$';
```

Comment:
Unrelated, but: a blast from the past!

Reply:
Ha, which bit!? The lodash template, jquery?

Reply:
I’m talking about the jQuery `$` alias. That dollar key got smashed quite a lot—especially if you were writing PHP on the back end, but that would have been _so uncool._